ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Node/Run.hs

```diff
@@ -106,6 +107,7 @@ class ( LedgerSupportsProtocol            blk
       , ShowProxy             (BlockQuery blk)
       , ShowProxy            (TxId (GenTx blk))
       , (forall fp. ShowQuery (BlockQuery blk fp))
+      , V2.LedgerSupportsV2LedgerDB blk
```

Comment:
The `V2.` prefix might be dropped, since the constraint name already includes `V2`. This also applies to the same qualified imports in the other modules in this PR